actions/user_actions.jsx

```diff
@@ -1,13 +1,10 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import React from 'react';
```

Comment:
Since this is removed, I wonder it the file should use .js extension instead of .jsx

Reply:
I'm removing a previous change, and before that change already was `.jsx`. We can do that change, but I think is not related to this PR.